projects/storefrontlib/src/cms-components/product/product-list/container/product-list-component.service.ts

```diff
@@ -7,37 +7,30 @@ import {
   ProductSearchPage,
   ProductSearchService,
   RoutingService,
-  SearchConfig,
 } from '@spartacus/core';
-import { combineLatest, Observable, Subscription } from 'rxjs';
+import { combineLatest, Observable } from 'rxjs';
 import {
   distinctUntilChanged,
   filter,
   pluck,
   shareReplay,
   tap,
 } from 'rxjs/operators';
-
-interface ProductListRouteParams {
-  brandCode?: string;
-  categoryCode?: string;
-  query?: string;
-}
-
-interface SearchCriteria {
-  currentPage?: number;
-  pageSize?: number;
-  sortCode?: string;
-  query?: string;
-}
-
+import { ProductListRouteParams, SearchCriteria } from './product-list.model';
+
+/**
+ * The `ProductListComponentService` is used to search products. The service is used
+ * on the Product Listing Page, for listing products and the facet navigation.
+ *
+ * The service exposes the product search results based on the category and search
+ * route parameters. The route parameters are used to query products by the help of
+ * the `ProductSearchService`.
+ */
 @Injectable({ providedIn: 'root' })
 export class ProductListComponentService {
   // TODO: make it configurable
   protected defaultPageSize = 10;
 
-  protected sub: Subscription;
```

Comment:
I think that removing this could be considered as a breaking change for clients that extended this class.